### PR TITLE
Add box sizing and padding to search section

### PR DESCRIPTION
**Pull request by Nikomus**
- Amend the search page to have total height 100%, rather than 100% + 2rem

https://trello.com/c/vc0BgKnd/1137-duplicate-scroll-bars

### CODE REVIEW

File: public/store/css/main.css [hunk 1]
```diff
@@ -726,10 +726,9 @@ SEARCH PAGE
 */
 
 .search {
-	padding-top: 2rem;
-	background-color: #FFF;
-	width: 80%;
-	margin-left: 10%;
+    background-color: #FFF;
```
Comment (alexalala) [inline public/store/css/main.css]: tabbyyyyy

Comment (Nikomus) [reply public/store/css/main.css]: Whattt. I think this is because I restored the code from a stash and the existing tabs space were lumpy.

Comment (alexalala) [reply public/store/css/main.css]: ah probably 😂 my text editor is awful, so might have been me.. 👻

**Comment by alexalala** [comment]
the keyframes still need some formatting luv <3

**Pull request merged**